core/src/main/java/org/infinispan/xsite/ClusteredCacheBackupReceiver.java

```diff
@@ -208,6 +207,10 @@ private XSiteStatePushCommand newStatePushCommand(List<XSiteState> stateList) {
       return commandsFactory.buildXSiteStatePushCommand(stateList.toArray(new XSiteState[0]), 0);
    }
 
+   private ClusteringDependentLogic getClusteringDependentLogic() {
+      return cache.getComponentRegistry().getComponent(ClusteringDependentLogic.class);
+   }
```

Comment:
Unless this method will be used more in follow up changes, this method seems unnecessary.

Reply:
I can remove it. It isn't used yet...